src/components/Photos.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Photo from './Photo';
+import PropTypes from 'prop-types';
+import styles from '../css/Photos.css';
+
+function Photos({ urls }) {
+  const listOfUrls = urls.map(url => {
+    return <li key={url}><Photo url={url} /></li>;
+  });
+
+  return (
+    <ul className={styles.ul}>
+      <li>{listOfUrls}</li>
```

Comment:
You're nesting `<li>`s here, which is no good. There's a warning in the browser console about this. 
```suggestion
      {listOfUrls}
```